app/v1/models/response/common/BusinessSourceSummary.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.common
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import v1.models.domain.TypeOfBusiness
+
+case class BusinessSourceSummary(typeOfBusiness: TypeOfBusiness,
+                                 selfEmploymentId: Option[String],
+                                 accountingPeriod: AccountingPeriodResponse,
+                                 bsasEntries: Seq[BSASEntries])
+
+object BusinessSourceSummary {
+
+  implicit val reads: Reads[BusinessSourceSummary] = (
+    (JsPath \ "incomeSourceType").read[TypeOfBusiness] and
```

Comment:
"incomeSourceType" is distinct from "typeOfBusiness" and is returned from DES as "01", "02", or "04". If you take a look at https://github.com/hmrc/individual-losses-api/blob/master/app/v1/models/des/IncomeSourceType.scala you'll see what you need to do (With TypeOfLoss replaced with TypeOfBusiness).

Probably end up looking something like:

    (JsPath \ "incomeSourceType").read[IncomeSourceType].map(_.toTypeOfBusiness) and